frontend/src/container/OrganizationSettings/AuthDomains/Edit/index.tsx

```diff
@@ -32,12 +37,24 @@ function EditSaml({
 				await onRecordUpdateHandler({
 					...record,
 					ssoEnabled: true,
-					samlConfig: {
-						...record.samlConfig,
-						samlCert: values.certificate,
-						samlEntity: values.entityId,
-						samlIdp: values.url,
-					},
+					ssoType: record.ssoType,
+					samlConfig:
```

Comment:
not sure what you mean but I made changes to this.

Reply:
we are returning a object based on `ssoType` so rather than writing the logic here only in component please make a wrapper or util function which will serve the purpose